plugins/bulk.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: bulk
+spec:
+  version: "v1.1.0-beta"
+  platforms:
+    - selector:
+       matchExpressions:
+       - {key: os, operator: In, values: [darwin, linux]}
+      uri: https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz
+      sha256: "80f7b55d82673253eb0957d2c248163fcce3e4c3bc609410b1e9ee2d6d6a91a7"
+      files:
+        - from: kubectl-plugins-*/kubectl-bulk
+          to: "."
+      bin: ./kubectl-bulk
+  shortDescription: Prints the environment variables.
+  caveats: |
+    Check documentation on
+    <!-- https://github.com/emreodabas/kubectl-plugins#kubectl-bulk -->
+    This plugin needs the following programs:
+    * grep
+    * sed
+  description: |
+    This plugin useful for Bulk operations.
+    You can easily do bulk operations on all resource types like deployments, services, pods etc.
```

Comment:
Instead of saying subjective statements like useful or easy, I recommend you clearly say what the plugins does, and perhaps provide an example.

Reply:
changed